.github/workflows/submit.yml

```diff
@@ -172,7 +172,7 @@ jobs:
       - name: Install dependencies
         run: |
           sudo apt-get update
-          sudo apt-get install gcc-10=10.2.0-5ubuntu1~20.04 g++-10=10.2.0-5ubuntu1~20.04 libxrandr-dev libxtst-dev libcups2-dev libasound2-dev
+          sudo apt-get install gcc-10=10.3.0-1ubuntu1~20.04 g++-10=10.3.0-1ubuntu1~20.04 libxrandr-dev libxtst-dev libcups2-dev libasound2-dev
```

Comment:
One dummy question: 
Why do we need to specify the real package name here? 
If weinstall gcc-10, I think apt system will pick up the latest gcc-10 for you.